vassal-app/src/main/java/VASSAL/build/module/DoActionButton.java

```diff
@@ -759,7 +759,9 @@ else if (LoopControl.LOOP_COUNTED.equals(loopType)) {
    */
   @Override
   public List<String> getFormattedStringList() {
-    return Arrays.asList(reportFormat.getFormat());
+    final List<String> l = new ArrayList<>();
+    l.add(reportFormat.getFormat());
+    return l;
```

Comment:
Why's this changing?

Reply:
Because it is now being over-ridden in SpecialDiceButton which attempts to add an additional entry to the returned list. Arrays.asList() returns an immutable list, so you get an UnsupportedOperation exception.